.github/workflows/test.yml

```diff
@@ -34,8 +34,7 @@ jobs:
       - name: Run tests
         run: docker-compose run test
```

Comment:
I considered changing it here based on a comment on another PR, I think, then decided it would be best to keep that as a separate effort. I think it would be best to solicit a go-ahead from the powers that be, and then make a concerted effort.